Assets/MRTK/Core/Utilities/Gltf/Serialization/GltfUtility.cs

```diff
@@ -170,13 +170,13 @@ public static GltfObject GetGltfObjectFromJson(string jsonString)
 
             if (gltfObject.extensionsRequired?.Length > 0)
             {
-                Debug.LogError($"Required Extension Unsupported: {gltfObject.extensionsRequired[0]}");
+                Debug.LogError("One or more unsupported glTF extensions required. Unable to load the model.");
                 return null;
             }
 
-            for (int i = 0; i < gltfObject.extensionsUsed?.Length; i++)
+            if (gltfObject.extensionsUsed?.Length > 0)
             {
-                Debug.LogWarning($"Unsupported Extension: {gltfObject.extensionsUsed[i]}");
+                Debug.Log("One or more unsupported glTF extensions in use, ignoring them.");
```

Comment:
For both of these, it still maybe feels worth printing out the "unsupported extensions". Otherwise, I might not know what to expect or how to fix it

Reply:
seems fair

Reply:
updated.